### DISCUSSION TITLE
Demo should contain XPath Examples

### DISCUSSION THREAD

**Discussion by pheonix-devapps** [FreshRSS/FreshRSS#5172] [Ideas]
I have check the demo website url added as feed but didn't found to have XPath Example. Can any suggest the development team to do so?

**Comment by BiscuitHub** [comment]
Hello,

If you're looking for a good example, there's a really good blog write up from a fellow named Dan Q.

https://danq.me/2022/09/27/freshrss-xpath/

It really helped me out when wrapping my smooth lizard brain around getting the scraper going.

PS: Killer feature FreshRSS team, 110% appreciated!

**Reply by Alkarex** [reply]
https://danq.me/2023/03/28/forward-comic-freshrss-xpath/
